### ISSUE TITLE
Align fixture data with real ID's

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#782]
**Is your feature request related to a problem? Please describe.**
Currently we have made up fixture data IDs. Now that we are integrating with the external dependencies we have realistic asset IDs. Currently the Cypress test use the article ID `c0000000025o` for development and production builds of the application. This restricts us from having dev and production builds set the RAZZLE_BASE_PATH differently #772 

**Describe the solution you'd like**
We could align the ID’s on test and fixture data meaning cypress can use a single asset ID when testing dev and production versions of the application. 

Also update the tables in https://github.com/BBC-News/simorgh/tree/latest/data and https://github.com/BBC-News/simorgh/issues/764#issuecomment-428518552 with these IDs

**Describe alternatives you've considered**
 We could keep the c0000000025o local ID’s and use override-able env vars to specify different assets at build time

**Testing notes**
The Cypress tests will need to be updated with the new article IDs -- this should be done by @jamesbrumpton -- the new article IDs will need to be accessible somewhere for this to happen.

Then:
The cypress tests pass on both development and production builds of the application.

**Additional context**
- c0000000025o: `c9rpqy7pmypo`
- c0000000027o: `c85pqyj5m2ko`
- c0000000028o: `cwv2xv848j5o`

**Comment by bcmn** [comment]
Is this blocked by #764?

**Comment by BogdanDogaru** [comment]
@bcmn This is not blocked by #764, as that issue is out of scope for now.
